Library/Homebrew/test/test_sandbox.rb

```diff
@@ -25,4 +25,21 @@ def test_deny_write
     end
     refute_predicate @file, :exist?
   end
+
+  def test_complains_on_failure
+    Utils.expects(:popen_read => "foo")
+    out, err = capture_io do
+      assert_raises(ErrorDuringExecution) { @sandbox.exec "false" }
+    end
+    assert_match out, "foo"
+  end
+
+  def test_ignores_bogus_python_error
+    bogus_error = "Mar 17 02:55:06 sandboxd[342]: Python(49765) deny file-write-unlink /System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/errors.pyc\n"
```

Comment:
While looking at this, discovered that all my `assert_match`es were backwards, oops. Agree that it improves the test~!